runtime/help/commands.md

```diff
@@ -36,6 +36,10 @@ quotes here but these are not necessary when entering the command in micro.
    A negative number can be passed to go inward from the end of the file.
    Example: -5 goes to the 5th-last line in the file.
 
+* `jump '[-]line[:col]'`: jumps with the given relative number from the current
```

Comment:
nit: maybe better "goes" instead of "jumps" (to make it clear that the only difference from `goto` is absolute vs relative).

Reply:
Also seems inconsistent that `jump` has `[-]line` while `goto` has just `line`.